falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.5.7
+
+* Add `alertmanager.extralabels` and `alertmanager.extraannotations` to the values,
+as a list of comma separated labels and annotations to add to the AlertManager trigger.
+* Add `smtp.autmechanism` and associated fields to be able to use any SASL mechanism.
+* Add `yandex.datastreams` and associated fields.
+* Add `nodered` and associated fields.
+* Add `mqtt` and associated fields.
+* Update `elasticsearch.type` default value to `_doc`
```

Comment:
Impressive, I created the PR less than 1h ago :+1:

Reply:
Hehe :smile: